crowd-ui/src/features/translationValidation/saga.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  call, put, takeEvery, SagaReturnType,
+} from 'redux-saga/effects';
+import { getSentenceFetch, setLoading, postUserData } from './slice';
+import { GET_SENTENCE_FETCH, POST_SENTENCE_DATA } from './action';
+import { fetchSentence, postUserSelection } from './api';
+
+type FetchUserResponseType = SagaReturnType<typeof fetchSentence>
+type PostUserResponseType = SagaReturnType<typeof postUserSelection>;
+
+function* GetSentenceFetch() {
+  const response: FetchUserResponseType = yield call(fetchSentence);
+  yield put(getSentenceFetch(response));
+  yield put(setLoading(true));
+  if (response.status >= 200 && response.status < 400) {
```

Comment:
why    200 <= status < 400 ?

Reply:
shouldn't be only 200 ?

Reply:
200 - 299 : [Successful responses](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#successful_responses)
300 - 399 : [Redirection messages](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#redirection_messages)